scripts/prepare_transport_data_input.py

```diff
@@ -95,14 +95,20 @@ def download_CO2_emissions():
     # Add ISO2 country code for each country
     CO2_emissions = CO2_emissions.rename(columns={"Country Name": "Country"})
     cc = coco.CountryConverter()
-    Country = pd.Series(CO2_emissions["Country"])
-    CO2_emissions["country"] = cc.pandas_convert(
-        series=Country, to="ISO2", not_found="not found"
+    CO2_emissions.loc[:, "country"] = cc.pandas_convert(
+        series=CO2_emissions["Country"], to="ISO2", not_found="not found"
     )
 
     # Drop region names that have no ISO2:
     CO2_emissions = CO2_emissions[CO2_emissions.country != "not found"]
 
+    # Drop region names where country column contains list of countries
+    CO2_emissions = CO2_emissions[
+        CO2_emissions.apply(
+            lambda x: False if not isinstance(x.country, str) else True, axis=1
+        )
+    ]
+
```

Comment:
```suggestion
    CO2_emissions = CO2_emissions[
        CO2_emissions.country.apply(lambda x: isinstance(x, str), axis=1)
    ]
```
Can you check if this passes the pre-commit?
Happy to merge right after :)

Great check!

Reply:
Thank you, @davide-f, for suggestion. I have made a minor change there. Just removed (axis=1). As inner part is now Series.